astropy/io/votable/exceptions.py

```diff
@@ -883,7 +883,7 @@ class W36(VOTableSpecWarning):
 class W37(UnimplementedWarning):
     """
     The 3 datatypes defined in the VOTable specification and supported by
-    ``astropy.io.votable`` are ``TABLEDATA``, ``BINARY`` and ``FITS``.
+    ``astropy.io.votable`` are ``TABLEDATA``, ``BINARY``, ``FITS``, ``PARQUET``.
```

Comment:
Parquet is not yet in the VOTable specification. It would be better to add a sentence that PARQUET serialization is a candidate for addition to the specification.

Reply:
can we add it since it is now "supported" (reading and writing VOTables with Parquet serialization)?
Also, to be precise: writing VOTables with FITS serialization is not supported. Only reading is...

Reply:
got it. Changed the documentation to:
"""
    The 3 datatypes defined in the VOTable specification and supported by
    ``astropy.io.votable`` are ``TABLEDATA``, ``BINARY``, and ``FITS``.
    In addition, ``astropy.io.votable`` also supports ``PARQUET`` serialization, which is 
    a candidate for addition to the VOTable specification.
    """